percona-server.80/ps-entry.sh

```diff
@@ -138,7 +138,7 @@ if [ "$1" = 'mysqld' -a -z "$wantHelp" ]; then
 
 		# install TokuDB engine
 		if [ -n "$INIT_TOKUDB" ]; then
-			ps-admin --docker --enable-tokudb -u root -p $MYSQL_ROOT_PASSWORD
+			ps-admin --docker --enable-tokudb -u root
```

Comment:
@yesqiao It seems that the same is needed for:
1. INIT_ROCKSDB init also (just few lines below your fix)
2. percona server 5.7 (I see the same logic there)
could you please update the PR.
Thanks!